### DISCUSSION TITLE
airpres is the sea-level pressure correct?

### DISCUSSION THREAD

**Discussion by NicWayand** [CH-Earth/summa#543] [Q&A] [answered]
airpres is the sea-level pressure correct?

**Comment by andywood** [comment, marked as answer]
l would use local pressure, which can be pretty different from SLP.  But
it's possible I've been doing this incorrectly if there's a
hardwired/automatic elevational adjustment in the code that I have
overlooked.  That might be a decent user option if it's not there, btw.

**Comment by wknoben** [comment]
I believe Andy is correct. The docs specify `airpres` should be at measurement height, which is defined as: 
```
the height (above bare ground) where the meteorological variables are specified
```

- https://summa.readthedocs.io/en/latest/input_output/SUMMA_input/#meteorological-forcing-files
- https://summa.readthedocs.io/en/latest/input_output/SUMMA_input/#forcing_file_measurement_height

There is a conversion function that converts sea level pressure into pressure at a specified elevation, but as far as I can tell that's not used (anymore?) in the rest of the code. 
https://github.com/CH-Earth/summa/blob/3ca07b723ee876394c11b8521081fede2c168b39/build/source/engine/conv_funcs.f90#L104-L114

**Reply by NicWayand** [reply]
Thanks Andy and Wouter!
